### ISSUE TITLE
Add curves for validation set in the feature selection plots

### ISSUE THREAD

**Issue by ch2ohch2oh** [ch2ohch2oh/belle_lambda#2]
With validation set also in the plot, we can visualize how overfitting changes with the number of features.
